### DISCUSSION TITLE
After reboot folders under storage are no longer accible with Permission denied error

### DISCUSSION THREAD

**Discussion by chribonn** [termux/termux-app#3816] [Q&A]
I have Termux installed on a Samsun SM-T290 table.  I notice that after a reboot the folders under **storage**, while visible all return _Permission denied_.

Performing an `ls-a` indicates that there are full access rights. Screen shot below. 

I am using the latest version.

Thanks

![photo_2024-02-01_17-46-14](https://github.com/termux/termux-app/assets/7222836/2172406a-2c4f-4357-b1a0-4fe9b322b41b)

**Comment by agnostic-apollo** [comment]
Disable storage permission in termux app info in android settings and enable it again
